rosbot_description/package.xml

```diff
@@ -15,13 +15,7 @@
   <url type="repository">https://github.com/husarion/rosbot_ros</url>
   <url type="bugtracker">https://github.com/husarion/rosbot_ros/issues</url>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
-
-  <build_depend>xacro</build_depend>
-  <exec_depend>xacro</exec_depend>
-  <exec_depend>ros_components_description</exec_depend>
-
   <export>
-    <build_type>ament_cmake</build_type>
+    <build_type>ament_python</build_type>
```

Comment:
If possible I think that it will be better to use ament_cmake for description also